disco_aws_automation/disco_elasticache.py

```diff
@@ -195,6 +197,9 @@ def _create_redis_cluster(self, cluster_name, engine_version, num_nodes, instanc
                                   not allowed for T1 and T2 instance types.
             domain_name (str): hosted zone id to use for Route53 domain name
             tags (List[dict]): list of tags to add to replication group
+            maintenance_window(string): specifies the weekly time range (of atleast 1 hour) in UTC during
+                                         which maintenance on the cache cluster is performed. Default 
```

Comment:
The default value is only if the user doesn't specify one. If there is no such field in the config, it will opt for the default value from within the function.